zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/identity/PermissionsBehavior.java

```diff
@@ -88,7 +93,31 @@ public Either<Rejection, AuthorizationRecord> permissionAlreadyExists(
                     currentResourceIds)));
       }
     }
+    return Either.right(record);
+  }
 
+  // TODO: we need to provide also the ownerType here when
```

Comment:
🤔 @nicpuppa we can already get the `ownerType` from the `AuthorizationRecord`. Or did I misunderstand something?

Reply:
Correct, but we still need to refactor the method to retrieve the permissions. It should take the ownerType and the ownerId as arguments, instead of the ownerKey